ledger/acctonline.go

```diff
@@ -251,7 +254,7 @@ func (ao *onlineAccounts) committedUpTo(committedRound basics.Round) (retRound,
 	defer ao.accountsMu.RUnlock()
 
 	retRound = basics.Round(0)
-	lookback = basics.Round(config.Consensus[ao.versions[len(ao.versions)-1]].MaxBalLookback)
+	lookback = basics.Round(ao.acctLookback)
```

Comment:
I'm not sure why online accounts tracker needs it

Reply:
If `acctonline` returned `0` for lookback here, I think it would have no effect since the largest lookback is chosen in `tracker.go`.